lib/inky/inky.ex

```diff
@@ -0,0 +1,625 @@
+defmodule Inky do
+  @moduledoc """
+  Documentation for Inky.
+  """
+
+  @doc """
+  Hello world.
+
+  ## Examples
+
+      iex> alias Inky
+      iex> state = Inky.setup(nil, :phat, :red)
+      iex> state = Enum.reduce(0..(state.height - 1), state, fn y, state ->Enum.reduce(0..(state.width - 1), state, fn x, state ->Inky.set_pixel(state, x, y, state.red)end)end)
+      iex> Inky.show(state)
+
+  """
+
+  alias Circuits.SPI
+  alias Circuits.GPIO
+  alias Inky.InkyPhat
+  alias Inky.InkyWhat
+  alias Inky.State
+  # alias Inky.Pixel
+
+  @reset_pin 27
+  @busy_pin 17
+  @dc_pin 22
+
+  # @mosi_pin 10
+  # @sclk_pin 11
```

Comment:
Do you still need this? If it's dead code then 💀💥🔫

Reply:
They were referenced in the original. I should double-check if they are used for anything there. But I don't believe so. In that case 🔪